run_dir/design/index.html

```diff
@@ -1,78 +1,82 @@
 {% extends "base.html" %}
 
 {% block stuff %}
-<h1 class="mt-4" id="scilifelabgenomics-status">SciLifeLab Genomics - Status</h1>
+<div class="container-xxl">
+  <div class="col-10 mb-5 mx-auto text-center">
+    <h1 class="mt-4 display-1">Genomics Status</h1>
+    <p class="lead fs-3">Status page for the Stockholm node of National Genomics Infrastructure</p>
 
-{% if not user %}
+    {% if not user %}
 
-<h2> Get Access </h2>
-<p>To get access to the resources on this site, you first need to log in with the account given to you by your manager.</p>
-<p><a class="btn btn-primary btn-lg", href="/login">Log in</a></p>
+    <h2> Get Access </h2>
+    <p>To get access to the resources on this site, you first need to log in with the account given to you by your manager.</p>
+    <p><a class="btn btn-primary btn-lg", href="/login">Log in</a></p>
 
-{% else %}
+    {% else %}
+    <p class="lead" id="updated-status">
+    </p>
+    <div class="mb-5 mt-3">
+      <a type="button" class="btn btn-primary btn-lg px-4 mr-3" href="/projects">Projects</a>
+      <a type="button" class="btn btn-primary btn-lg px-4 mr-3" href="/flowcells">Flowcells</a>
+      <a type="button" class="btn btn-primary btn-lg px-4 mr-3" href="/worksets">Worksets</a>
+      <a type="button" class="btn btn-primary btn-lg px-4" href="/deliveries">Deliveries</a>
+    </div>
+    <div class="col-8 fs-4 mx-auto" id="sensorpush_status">
+      <div class="spinner-border" role="status">
+        <span class="visually-hidden">Loading...</span>
+      </div>
+    </div>
+    <p class="lead fs-3 mt-5">Want to see something different here? Let us know on the
+      <a class="text-decoration-none" href="/suggestion_box">suggestions page.</a>
+    </p>
 
-<p>Genomics status displays information from multiple data sources. <span id="updated-status"></span></p>
+  </div>
 
-<div class="alert alert-info">
-    <p class="mb-0" style="text-align:center;">Want to see something different here? Let us know on the
-      <a class="text-decoration-none" href="/suggestion_box">suggestions page</a>! We're currently collecting feedback on
-      how the homepage should look...</p>
-</div>
+  <div class="row pt-5">
+    <div class="col-lg-7">
+      <h3>Longest waiting projects</h3>
```

Comment:
As Sofia said, this table seems to be a bit long.

Reply:
I don't think it's actually longer than before, I made a larger part of it visisble before the scroll starts so that the footer is not on the middle of the screen on larger screens.

Reply:
I think we will remove it eventually so if it's ok, I say we don't change it right now.